demo/scripts/controls/sidePane/editorOptions/ExperimentalFeatures.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+import BuildInPluginState from '../../BuildInPluginState';
+import { ExperimentalFeatures } from 'roosterjs-editor-types';
+
+export interface ExperimentalFeaturesProps {
+    state: ExperimentalFeatures[];
+    resetState: (callback: (state: BuildInPluginState) => void, resetEditor: boolean) => void;
+}
+
+const FeatureName = {
+    [ExperimentalFeatures.ListChain]: 'Enable List Chain for numbering list',
+    [ExperimentalFeatures.MergePastedLine]:
+        'When paste, try merge pasted content to the same line with existing content',
+    [ExperimentalFeatures.NewBullet]: 'Toggle bullet using VList',
+    [ExperimentalFeatures.NewIndentation]: 'Toggle indentation using VList',
+    [ExperimentalFeatures.NewNumbering]: 'Toggle numbering using VList',
+};
+
+export default class ExperimentalFeaturesPane extends React.Component<
+    ExperimentalFeaturesProps,
+    {}
+> {
+    render() {
+        return (
+            <>
+                {[
+                    ExperimentalFeatures.ListChain,
+                    ExperimentalFeatures.MergePastedLine,
+                    ExperimentalFeatures.NewBullet,
```

Comment:
This can be replaced with Object.keys(FeatureName).map(name => this.renderFeature(name))